### DISCUSSION TITLE
Search by barcode?

### DISCUSSION THREAD

**Discussion by ghost** [joalla/discogs_client#78] [Support] [answered]
Hi,

I'm hoping to be able to find releases by inputting the barcode. Is this possible within this Python package?

Thanks!

**Comment by AnssiAhola** [comment, marked as answer]
Hi @DeputyBeagle 
Yes you can, with optional search parameters like so
```python
client.search(barcode="<barcode_here>")
``` 
All supported parameters are listed here https://www.discogs.com/developers/#page:database,header:database-search-get
Link to our documentation for more details https://python3-discogs-client.readthedocs.io/en/latest/quickstart.html#searching
